Fix Align VRMS logo

bconti123 opened the pull request:
Fixes #1917 

### What changes did you make and why did you make them ?

  - I added `@media (max-width:450px)` for the mobile screen on VRMS logos aligned, because the changing padding on the desktop platform will affect the mobile platform.
  - Changed padding to `padding: 3vh` for desktop and `padding: 1vh 1vh 3vh 1vh` for mobile.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->

<details>
<summary>Visuals before changes are applied</summary>

![OGvrms](https://github.com/user-attachments/assets/1ecc0fe5-33dc-4ce1-bd2f-84d336f294ac)


</details>

<details>
<summary>Visuals after changes are applied</summary>

<img src="https://github.com/user-attachments/assets/8a6ad3cf-6bf0-4da3-87fc-9a70b7c3e5e7" alt="desktop">
<img src="https://github.com/user-attachments/assets/33514667-8608-442a-80eb-2b2278c917a6" alt="mobile" width="450px">    

</details>

bconti123 committed:
fix-align-VRMS-logo

bconti123 committed:
Merge branch 'development' into 1917-align-VRMS-logo

JackHaeg committed:
Merge branch 'development' into 1917-align-VRMS-logo

trillium commented:
![image](https://github.com/user-attachments/assets/3e139171-5ebe-4b15-992e-e4fb62ffd8ee)
![image](https://github.com/user-attachments/assets/365830c0-61ec-4608-9b03-7d5bbd72d4d8)

Discussing these screenshots on the call tongiht

trillium reviewed (changes requested):
<img width="550" alt="image" src="https://github.com/user-attachments/assets/564b72db-51b3-4f30-9c19-352e61d360e7" />


Hey Bryant, thanks for working on this!

Can we get the two divs on the page to have the same level of distance from the left edge?

If you need to add an internal div on the page feel free to, doesn't need to only be changes to the CSS file.

If possible as well I'd prefer if we steer away from using vh measurements for padding becuase of the way they vary. This might have been there already, feel free to clean up and change any padding/margin based vh units if you see them

bconti123 commented:
@trillium, Thank you for your feedback. I will work on and look into improving the responsive design.

bconti123 committed:
Fixed Align

bconti123 commented:
@trillium 

I made the two divs have similar CSS (`home-headers` and `home-bottons`), and the alignment looks better. I removed padding and added margin-top and margin-bottom in `home-headers` from `home.scss`.

Let me know if any changes are needed.

trillium committed:
Merge branch 'development' into 1917-align-VRMS-logo

JackHaeg committed:
Merge branch 'development' into 1917-align-VRMS-logo

JackHaeg commented:
Hi @trillium! Can you please take a look at the updates @bconti123 made when you have a chance?

trillium commented:
Hey hey, from the call today, would you mind making sure that the container width doesnt reduce as a result of this PR?

Thanks :)

trillium commented:
<img width="517" alt="image" src="https://github.com/user-attachments/assets/fb2ca5ba-0d08-477b-9e70-6e49c48ae679" />

As per chat, lets have the outside container establish boundaries for the content

Please use `em` units

bconti123 committed:
Merge branch 'development' into 1917-align-VRMS-logo

bconti123 committed:
Change Margin to Padding and vh units to 3em following the issue's description.

bconti123 commented:
Hi @trillium, I updated the layout and replaced vh units with em units. I think I might have misunderstood part of what we discussed about the container in the previous Zoom meeting last week. I want to confirm. Is that yellow line outside the container we discussed? See screenshot below:

<details>
<summary>Yellow Line outside the app container</summary>

![LogoPaddingShow](https://github.com/user-attachments/assets/7cc4e192-93b5-4d5d-940f-702c3da1fac2)

</details>

I would like to discuss the app container and share my screen in the meeting tonight.
Let me know what I should fix ahead of time. Thank you!

trillium commented:
Hey Bryant, sorry to take so long here.

I feel like I've overcomplicated this. It's fine as it is now, happy to merge

trillium committed:
Merge branch 'development' into 1917-align-VRMS-logo
